app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnitsMap/styles.module.css

```diff
@@ -1,5 +1,32 @@
 .national-society-local-units-map {
+    .content {
+        position: relative;
 
+        .map-detail {
```

Comment:
Let's try to avoid any contents inside the map. Let's discuss if we can add these like legend in other places.